tests/Cucumber/extra_testdata/good/language_tag_after_feature_started.feature

```diff
@@ -0,0 +1,4 @@
+Feature: Some feature
+# language: ru
+
+    Given something wrong
```

Comment:
This test case where `Given something wrong` is actually part of the description rather than being a step is confusing IMO. I suggest adding a `Scenario:` line. Otherwise, this test would _also_ be covering the weird edge case of "step keyword in feature description" that has its dedicated test. It makes tests more reliable (even more important for bad tests actually, as our CompatibilityTest does not guarantee that we fail for the same reason than cucumber as they accept any failure)

Reply:
same for all other test cases that are not about "step keyword in feature description" IMO

Reply:
Good shout. These are all based on the structures we had in the existing ParserExceptions examples but I agree they're confusing. Will fix.